src/irmin-pack/pack_index.ml

```diff
@@ -80,7 +82,10 @@ module Make (K : Irmin.Hash.S) = struct
 
   let v = Index.v ~cache
 
-  let add t k v = replace t k v
+  let add ?overcommit t k v =
+    match overcommit with
+    | Some true -> replace ~overcommit:() t k v
+    | _ -> replace t k v
```

Comment:
Perhaps both `overcommit` options should be unified to avoid this?

Reply:
Done in https://github.com/mirage/irmin/pull/1183/files#diff-a8e8d436cf37ad285558a295e2e031ed0ecd868fe9490b411bbe5eefd3c9186dR85